experiments/calibration/global_bucket/calibrate_bucket.jl

```diff
@@ -0,0 +1,72 @@
+ENV["JULIA_WORKER_TIMEOUT"] = "300.0"
+
+using ClimaLand
+dir = pkgdir(ClimaLand)
+import ClimaCalibrate: forward_model, parameter_path, path_to_ensemble_member
+import ClimaCalibrate as CAL
+using Distributed
+addprocs(
+    CAL.PBSManager(2),
+    q = "main", # not prio
+    A = "UCIT0011",
+    l_select = "ngpus=4",
```

Comment:
```suggestion
    l_select = "ngpus=1:ncpus=4",
```
I think you only need one GPU per task, but I may be mistaken.